Removes an unnecessary mapping operation.

ndouglas opened the pull request:
## Description
Looks like a find-and-replace left a little bit of cruft behind.

## Testing done


## Screenshots


## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

mpelzsherman reviewed (commented):
Please confirm this doesn’t break the breadcrumbs on the health-services pages, e.g. https://www.va.gov/pittsburgh-health-care/health-services/

ndouglas commented:
Can you think of a case in which `someArray.map((value) => (value === 'health-services' ? 'health-services' : value))` is not equivalent to `someArray`?  Or `value === 'health-services' ? 'health-services' : value` is not equal to `value`?

I'd prefer _not_ to break the website, and I'm always open to the possibility that I'm a walking apocalypse who can trigger an exception or edge case in even the simplest code, but I just spent some time trying to break this assumption in a JSFiddle and haven't had any luck.

```javascript
const pathStrings = [
  'health-services',
  'totally-not-health-services',
  '/health-services',
  'health-services/',
  'three-health-services-stacked-in-a-trenchcoat',
  'i-see-a-little-silhouetto-of-a-health-serviceo',
  'im-just-a-poor-health-serviceo-nobody-loves-me',
];

function arrayEquals(a, b) {
  if (a === b) { return true; }
  if (a == null || b == null) { return false; }
  if (a.length !== b.length) { return false; }
  for (var i = 0; i < a.length; ++i) {
    if (a[i] !== b[i]) { return false; }
  }
  return true;
}

var failed = false;
for (var pathString of pathStrings) {
  const pathArray = pathString.split('/').map(value => (value === 'health-services' ? 'health-services' : value));
  const pathArray2 = pathString.split('/');
  if (!arrayEquals(pathArray, pathArray2)) {
    console.error(pathArray, pathArray2);
    failed = true;
  }
}

console.log('The test ' + (failed ? 'did' : 'did not') + ' fail.');
```

mpelzsherman commented:
I agree - seems safe to remove. Just check the review instance to be sure.

On Fri, Jun 18, 2021 at 10:05 AM Nathan Douglas ***@***.***>
wrote:

> Can you think of a case in which someArray.map((value) => (value ===
> 'health-services' ? 'health-services' : value)) is not equivalent to
> someArray? Or value === 'health-services' ? 'health-services' : value is
> not equal to value?
>
> I'd prefer *not* to break the website, and I'm always open to the
> possibility that I'm a walking apocalypse who can trigger an exception or
> edge case in even the simplest code, but I just spent some time trying to
> break this assumption in a JSFiddle and haven't had any luck.
>
> const pathStrings = [
>   'health-services',
>   'totally-not-health-services',
>   '/health-services',
>   'health-services/',
>   'three-health-services-stacked-in-a-trenchcoat',
>   'i-see-a-little-silhouetto-of-a-health-serviceo',
>   'im-just-a-poor-health-serviceo-nobody-loves-me',];
> function arrayEquals(a, b) {
>   if (a === b) { return true; }
>   if (a == null || b == null) { return false; }
>   if (a.length !== b.length) { return false; }
>   for (var i = 0; i < a.length; ++i) {
>     if (a[i] !== b[i]) { return false; }
>   }
>   return true;}
> var failed = false;for (var pathString of pathStrings) {
>   const pathArray = pathString.split('/').map(value => (value === 'health-services' ? 'health-services' : value));
>   const pathArray2 = pathString.split('/');
>   if (!arrayEquals(pathArray, pathArray2)) {
>     console.error(pathArray, pathArray2);
>     failed = true;
>   }}
> console.log('The test ' + (failed ? 'did' : 'did not') + ' fail.');
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/department-of-veterans-affairs/content-build/pull/271#issuecomment-864063868>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAATTCZEFAWDHI5PI7VGXFDTTNHD7ANCNFSM465V6GDA>
> .
>
-- 

Michael Pelz-Sherman
***@***.***
919-279-0825

mpelzsherman commented:
You shouldn't need to build from Tugboat - our review instances normally do a build you can check.
Unfortunately there's an issue with the review instance deploy links not showing up on Github.
I think @timwright12 et al are looking into this. I found the review instance [link](http://4087c08dbf509237c4104411d25a6c89.review.vetsgov-internal/) on [jenkins](http://jenkins.vfs.va.gov/blue/organizations/jenkins/deploys%2Fvets-review-instance-deploy/detail/vets-review-instance-deploy/45988/pipeline), but it's not working - which is also an issue hopefully someone is looking into?

The test cases you came up with are certainly creative, but when I see some weird code like this, I usually look at the git history to see who wrote it and why.

In this case, I found this [commit](https://github.com/department-of-veterans-affairs/content-build/commit/2cd6340f437fec0a1f3468fec4f6cc72dbd053b4#diff-fe834849f52ec1373839c7ce71413a87ecdd0481db355f53722f00323fd35371L145-L147). It looks like @ethanteague changed the function - it used to convert `health-services` to `our-health-services` but at some point that was deemed unnecessary.

Anyway - I think your change is fine. Thanks for cleaning up the cruft. There's probably a lot more where that came from! 😄
